backend/src/main/java/gov/cdc/usds/simplereport/validators/PatientBulkUploadFileValidator.java

```diff
@@ -39,35 +40,91 @@ public List<FeedbackMessage> validate(InputStream csvStream) {
-      ValueOrError street = getValue(row, "Street", true);
-      ValueOrError street2 = getValue(row, "Street2", false);
-      ValueOrError city = getValue(row, "City", false);
-      ValueOrError county = getValue(row, "County", false);
-      ValueOrError state = getValue(row, "State", true);
-      ValueOrError zipCode = getValue(row, "ZipCode", true);
-      ValueOrError phoneNumber = getValue(row, "PhoneNumber", true);
-      ValueOrError phoneNumberType = getValue(row, "PhoneNumberType", false);
-      ValueOrError employedInHealthcare = getValue(row, "employedInHealthcare", true);
-      ValueOrError residentCongregateSetting = getValue(row, "residentCongregateSetting", true);
-      ValueOrError role = getValue(row, "Role", false);
-      ValueOrError email = getValue(row, "Email", false);
+      PatientUploadRow extractedData = new PatientUploadRow(row);
 
+      errors.addAll(extractedData.validateHeaders());
+
+      // validate individual values
+
+      // demographics
+      errors.addAll(validateDate(extractedData.dateOfBirth));
+      errors.addAll(validateRace(extractedData.race));
+      errors.addAll(validateBiologicalSex(extractedData.biologicalSex));
+      errors.addAll(validateEthnicity(extractedData.ethnicity));
+
+      // housing, work, and role
+      errors.addAll(validateYesNoAnswer(extractedData.residentCongregateSetting));
+      errors.addAll(validateYesNoAnswer(extractedData.employedInHealthcare));
+      errors.addAll(validateRole(extractedData.role));
+
+      // address
+      errors.addAll(validateState(extractedData.state));
+      errors.addAll(validateZipCode(extractedData.zipCode));
+
+      // contact info
+      errors.addAll(validatePhoneNumber(extractedData.phoneNumber));
+      errors.addAll(validatePhoneNumberType(extractedData.phoneNumberType));
+      errors.addAll(validateEmail(extractedData.email));
+    }
+
+    return errors;
+  }
+
+  @Getter
+  public static class PatientUploadRow {
+    private final ValueOrError firstName;
+    private final ValueOrError lastName;
+    private final ValueOrError middleName;
+    private final ValueOrError suffix;
+    private final ValueOrError race;
+    private final ValueOrError dateOfBirth;
+    private final ValueOrError biologicalSex;
+    private final ValueOrError ethnicity;
+    private final ValueOrError street;
+    private final ValueOrError street2;
+    private final ValueOrError city;
+    private final ValueOrError county;
+    private final ValueOrError state;
+    private final ValueOrError zipCode;
+    private final ValueOrError phoneNumber;
+    private final ValueOrError phoneNumberType;
+    private final ValueOrError employedInHealthcare;
+    private final ValueOrError residentCongregateSetting;
+    private final ValueOrError role;
+    private final ValueOrError email;
+
+    public PatientUploadRow(Map<String, String> rawRow) {
+      firstName = getValue(rawRow, "first_name", true);
```

Comment:
note that our headers have changed and are more consistent on the suggestion of @lauraykerr: "first_name" instead of "firstName"